KUBEVIRT_LOCAL_TESTING.md

```diff
@@ -9,15 +9,15 @@ Steps:
 ```bash
 cd .../kubevirtci
 cd cluster-provision/k8s/1.17.0
-./provision.sh      # this also calls cluster-up to check whether the cluster will really start and have the pods ready
+../provision.sh      # this also calls cluster-up to check whether the cluster will really start and have the pods ready
 ```
 
 ### kubevirt: prepare for tests
 
 ```bash
 # set local provision test flag
 export KUBEVIRTCI_PROVISION_CHECK=1
-# sync changes to kubevirt cluster-up
+# sync changes to kubevirt cluster-up, notably cluster/images.sh
```

Comment:
I guess that we still have to document that cluster/images.sh needs to be manually updated, but a aprt from that looks good.